examples/seismic/viscoacoustic/operators.py

```diff
@@ -63,7 +62,85 @@ def blanch_symes(model, geometry, v, p, **kwargs):
     return [u_v, u_r, u_p]
 
 
-def ren(model, geometry, v, p, **kwargs):
+def sls_2nd_stencil(model, geometry, v, p, **kwargs):
+    """
+    Stencil created from Bai (2014) viscoacoustic wave equation.
```

Comment:
And what exactly do you mean by 'stencil' here? Would something along the lines of 'Implementation of the 2nd order viscoacoustic wave-equation from Bai (2014)' be better?

Reply:
Yes. We agree. We will change.